dace/runtime/include/dace/types.h

```diff
@@ -37,6 +37,7 @@
     #define DACE_HDFI inline
     #define DACE_HFI inline
     #define DACE_DFI inline
+    typedef short half;
```

Comment:
`short` != `float16`, what's the idea here?

Reply:
This also seems to be what's failing the tests.

Reply:
CPU/C++ code does not (yet) natively support half-precision floating point. Since CUDA and NumPy do, we need a way to communicate that. In CUDA, `float16` is defined as `short` in certain points of host code, so this fits. Now the tests will hopefully pass.